OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -1673,18 +1674,28 @@ def test_browse_filtering_allows_good(client, mockdata, session):
             follow_redirects=True,
         )
 
-        filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Rank</dt>")[1:]
+        def normalize_tokens_for_comparison(html_str: str, split_str: str):
+            """Remove new lines, leading, and closing spaces between <dd> elements in
+            formatted HTML."""
+            parsed_list = html_str.data.decode(ENCODING_UTF_8).split(split_str)[1:]
+            parsed_list = [re.sub(r"<dd>\n\s+", "<dd>", token) for token in parsed_list]
+            parsed_list = [
+                re.sub(r"\n\s+</dd>", "</dd>", token) for token in parsed_list
+            ]
+            return parsed_list
```

Comment:
This addresses the problem that with the newly formatted HTML have a starting line break and A LOT of spaces between the `<dd>` elements.